R/mutate.R

```diff
@@ -84,19 +86,25 @@
 #'
 #' # The difference:
 #' tidy(qq_rec, number = 1)
-step_mutate <- function(recipe, ...,
+step_mutate <- function(recipe, 
+                        ...,
+                        .pkgs = character(),
```

Comment:
since it is after the dots, order does matter, and for me I think having the arguments the user is more likely to need early makes sense

Reply:
Agreed.